### PR TITLE
Magnetic top panel

### PR DESCRIPTION
**Pull request by Printopal**
* [x ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x ] Submitted STL files are oriented and scaled properly for printing.
  * [x ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
This modification adds new STLs/OBJs for printing magnet mounts and panel skirts to build the full magnetic top panel for the 1.8 and 2.4 series 250/300/350. There are additional images and a readme for description
#### How was it tested? 
Printed and assembled. Video included.
Not tested with glass, and not recommended
#### Any background context you want to provide?
Allow quick access from top of the printer
#### Screenshots (if appropriate)
Screenshots of the BOM, 3D printing chart and assembly
#### Further notes
For standard acrylic panel sheet or acrylic sheet with abrasion resistant coating.

### CODE REVIEW

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

I haven't reviewed everything yet, but some general comments:

- Please replace OBJ files with STL
- Why do you have multiple versions of each size STLs (220, 221, 222), (60, 61, 62, 63, 64)?
- Can you simplify all the frame_corner pieces so that there's only one size that works for everyone?  Are you trying to make them press fit into different extrusions?  Can you instead just use a screw to attach them to the extrusion and the not worry about such exact fit?

**Comment by Printopal** [comment]
@KiloQubit thanks for your feedback
Now there are only STL files.
Originally I have included multiples version of lengths and holes diameter to be sure everybody get stuff ready to use out of 3D printing. Frame corner pieces is now a single STL, the actual file don't allow to screw the corner into the extrusion because of it thickness / screw head height. The basic idea is to get all the magnetic panel as thin as possible. A screw-on alternative is absolutely possible but the full assembly will be a bit higher and will need another foam reference (higher too).
The frame corner file I left is press fit to the extrusion, in the worst case it can be sanded or glued but it's not something that I like.
I won't be available for next two weeks, just know I will do the necessary to complete your requests.

**Comment by KiloQubit** [comment]
> @KiloQubit thanks for your feedback
> Now there are only STL files.
> Originally I have included multiples version of lengths and holes diameter to be sure everybody get stuff ready to use out of 3D printing. Frame corner pieces is now a single STL, the actual file don't allow to screw the corner into the extrusion because of it thickness / screw head height. The basic idea is to get all the magnetic panel as thin as possible. A screw-on alternative is absolutely possible but the full assembly will be a bit higher and will need another foam reference (higher too).
> The frame corner file I left is press fit to the extrusion, in the worst case it can be sanded or glued but it's not something that I like.
> I won't be available for next two weeks, just know I will do the necessary to complete your requests.

You haven't actually updated the PR with the changes you mention in your comment.

**Review by Printopal — COMMENTED** [review]
Removing OBJ files
Removing multiple sizes versions

File: printer_mods/Printopal/Magnetic_top_panel/readme.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Acrylic sheet with abrasion resistant coating may be considered for V1.8 to prevent micro scratch caused by the reverse bowden tube. This may also appear with full height prints on V2.4.
+I don’t recommend using glass plate because of its weight, this may result in broken top skirt and broken glass!
+
+
+# Instructions
+1. Insert magnets into panel corner and frame corner respecting their polarity.
+2. Insert frame corner on top of the 4 vertical 2020 extrusions.
+3. Put panel corner on frame corner, sides should be flush.
+4. Glue the acrylic panel sheet on panel corners, be sure to keep panel corner and frame corner sides flush while gluing the acrylic panel.
```
Comment (KiloQubit) [inline printer_mods/Printopal/Magnetic_top_panel/readme.md:33]: What kind of glue?  Or would VHB tape work?  Same questions for steps 6 and 7.

File: printer_mods/Printopal/Magnetic_top_panel/readme.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+5. Remove the panel from the frame, adjust foam length and glue it under the acrylic panel sheet (same side as panel corner).
+6. Glue top corner skirt on the acrylic panel sheet.
+7. Adjust top skirt length and glue on the acrylic panel sheet. You can easily adjust the length in your slicer.
+#
+![Magnetic_top_panel](https://user-images.githubusercontent.com/85416803/120905966-53f65000-c61b-11eb-8367-2481f096431a.jpg)
```
Comment (KiloQubit) [inline printer_mods/Printopal/Magnetic_top_panel/readme.md:38]: Can you upload all the images from this README into a directory under your mod (maybe `printer_mods/Printopal/Magnetic_top_panel/images/`) and change the references in this README to use those uploaded images?  For example, if this image was uploaded as `images/panel.jpg`, then your image would be

    Magnetic_top_panel

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the update!  I just have some README comments below.

File: printer_mods/Printopal/Magnetic_top_panel/readme.md (hunk 2 above)
Comment (Printopal) [reply printer_mods/Printopal/Magnetic_top_panel/readme.md]: Thank for your feedback! I created an Images folder and modified their path in README as you suggested.
I couldn't get the same thing working for the video, any advices?

Comment (KiloQubit) [reply printer_mods/Printopal/Magnetic_top_panel/readme.md]: I could not find a way to do this with a video.  So you can either leave it as you have it now, or upload it to the directory and just link to it - but then it won't play embedded, it'll just make the user download it.

Can you also comment on my question about glue/VHB tape?

File: printer_mods/Printopal/Magnetic_top_panel/readme.md (hunk 1 above)
Comment (Printopal) [reply printer_mods/Printopal/Magnetic_top_panel/readme.md:33]: Standard liquid superglue is used here.
WHB will add an extra heigth that will cancel the foam seal.

File: printer_mods/Printopal/Magnetic_top_panel/readme.md (hunk 2 above)
Comment (Printopal) [reply printer_mods/Printopal/Magnetic_top_panel/readme.md]: Thanks, I will let the video like this so.
I missed that question sorry, it's all done now.

File: printer_mods/Printopal/Magnetic_top_panel/readme.md (hunk 1 above)
Comment (KiloQubit) [reply printer_mods/Printopal/Magnetic_top_panel/readme.md:33]: Can you add that note to the README to make sure someone won't try to use VHB tape?

Comment (Printopal) [reply printer_mods/Printopal/Magnetic_top_panel/readme.md]: Sure, it's updated

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**